docker-compose.yml

```
@@ -47,4 +47,4 @@ networks:
       driver: default
       config:
         - subnet: 10.42.42.0/24
-        - subnet: fd00::0/112
+        - subnet: fdcc:ad94:bacf:61a4::/64
```

To avoid address collisions I would change the prefix of the internal docker network here to the following:
```suggestion
        - subnet: fdcc:ad94:bacf:61a3::/64
```